apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -24,15 +25,17 @@ export function toResponseWorkflowDto(
     user: preferences?.source[PreferencesTypeEnum.USER_WORKFLOW] || null,
     default: preferences?.source[PreferencesTypeEnum.WORKFLOW_RESOURCE] || DEFAULT_WORKFLOW_PREFERENCES,
   };
+  const workflowName = template.name || 'Missing Name';
 
   return {
     _id: template._id,
+    slug: `${slugifyName(workflowName)}_${encodeBase62(template._id)}`,
```

Comment:
self note: add the entity name  `-wf-`  before the id so that it is easy to support backward compatibility with these ids once we shorten them